src/lib/es2022.error.d.ts

```diff
@@ -1,8 +1,55 @@
 interface ErrorOptions {
-  cause?: Error;
+    cause?: Error;

```

Comment:
`cause` should be typed as `unknown` (or `any` in a less strict way) as the value being thrown in JavaScript can be arbitrary values.